README.md

```diff
@@ -1,3 +1,6 @@
+# git & git hub
+## I'm making a pool request for the second time, I hope I did everything right this time
+
 # Kottans-frontend
```

Comment:
Nicely done.
The only request - can you move your changes(lines) under those ones - that points to the Discord with the help chat...?